### ISSUE TITLE
location of code about your text prior architecture

### ISSUE THREAD

**Issue by HellbroWDR** [mjq11302010044/TATT#16]
Can you point where is the location of your text prior architecture in your code, please?
I really want to know how your architecture use the output of CRNN as text prior.
It is hard to find it.

And another question, does it seem that your code don't use text prior in testing?

**Comment by llvhjgtdl** [comment]
请问你解决了吗，我也找不到

**Comment by HellbroWDR** [comment]
解決了，它的text prior生成與轉換的部分在interfaces/super_resolution.py裡面
CRNN產生出來的sequence probability經過softmax轉換成總和為1的tensor之後再squeeze，就是text prior

**Comment by llvhjgtdl** [comment]
谢谢你的回复呀，但是你可以具体说一下text prior 的生成与转换部分在interfaces/super_resolution.py里面的哪部分代码吗，我是小白一个有很多不懂得地方，如有打扰还请见谅。



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年9月27日(星期三) 晚上7:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mjq11302010044/TATT] location of code about your text prior architecture (Issue #16)





 
解決了，它的text prior生成與轉換的部分在interfaces/super_resolution.py裡面
 CRNN產生出來的sequence probability經過softmax轉換成總和為1的tensor之後再squeeze，就是text prior

**Comment by ych-tju** [comment]
> 谢谢你的回复呀，但是你可以具体说一下text prior 的生成与转换部分在interfaces/super_resolution.py里面的哪部分代码吗，我是小白一个有很多不懂得地方，如有打扰还请见谅。
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: ***@***.***&gt;; 发送时间: 2023年9月27日(星期三) 晚上7:44 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; ***@***.***&gt;; 主题: Re: [mjq11302010044/TATT] location of code about your text prior architecture (Issue #16) 解決了，它的text prior生成與轉換的部分在interfaces/super_resolution.py裡面 CRNN產生出來的sequence probability經過softmax轉換成總和為1的tensor之後再squeeze，就是text prior — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***&gt;

我也是小白，这代码我看了很久很久，我感觉应该是在这一行，位于interfaces/super_resolution里面：
cascade_images, pr_weights = model_list[pick](images_lr if not self.args.for_cascading else cascade_images,
                                                              label_vecs_final.detach())
它传入两个参数，一个是image，一个是产生的text prior：label_vecs_final.detach()。我也不知道我理解的对不对，你可以参考一下

**Comment by llvhjgtdl** [comment]
应该是这样，谢谢





------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月10日(星期天) 上午10:52
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mjq11302010044/TATT] location of code about your text prior architecture (Issue #16)





  
谢谢你的回复呀，但是你可以具体说一下text prior 的生成与转换部分在interfaces/super_resolution.py里面的哪部分代码吗，我是小白一个有很多不懂得地方，如有打扰还请见谅。
 …
 ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: @.&gt;; 发送时间: 2023年9月27日(星期三) 晚上7:44 收件人: @.&gt;; 抄送: @.&gt;; @.&gt;; 主题: Re: [mjq11302010044/TATT] location of code about your text prior architecture (Issue #16) 解決了，它的text prior生成與轉換的部分在interfaces/super_resolution.py裡面 CRNN產生出來的sequence probability經過softmax轉換成總和為1的tensor之後再squeeze，就是text prior — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: @.***&gt;
  
我也是小白，这代码我看了很久很久，我感觉应该是在这一行，位于interfaces/super_resolution里面：
 cascade_images, pr_weights = model_list[pick](images_lr if not self.args.for_cascading else cascade_images,
 label_vecs_final.detach())
 它传入两个参数，一个是image，一个是产生的text prior：label_vecs_final.detach()。我也不知道我理解的对不对，你可以参考一下
